www/src/app/views/InboxContainer.js

```diff
@@ -30,114 +20,74 @@ export default class InboxContainer extends Component {
 
     this.handleResolve = this.handleResolve.bind(this);
     this.handleArchive = this.handleArchive.bind(this);
+
+    this.state = {
+      editText: '',
```

Comment:
Ah, cool, this is a good idea.  It allows us to not keep both the old text and the new text on the `update` while the editing input is active.